build.sbt

```diff
@@ -555,6 +555,16 @@ lazy val finagleBackend = (projectMatrix in file("finagle-backend"))
   .jvmPlatform(scalaVersions = scala2)
   .dependsOn(core % compileAndTest)
 
+lazy val armeriaBackend = (projectMatrix in file("armeria-backend"))
```

Comment:
the project also needs to be added to root aggregates, so that it's included when running tests from root

Reply:
added